package-lock.json

```diff
@@ -3352,6 +3352,16 @@
       "resolved": "https://registry.npmjs.org/@types/history/-/history-4.7.3.tgz",
       "integrity": "sha512-cS5owqtwzLN5kY+l+KgKdRJ/Cee8tlmQoGQuIE9tWnSmS3JMKzmxo2HIAk2wODMifGwO20d62xZQLYz+RLfXmw=="
     },
+    "@types/hoist-non-react-statics": {
```

Comment:
All looks good to me except to revert this change since no change in `package.json`.